build/release-pipeline.yml

```diff
@@ -454,97 +449,117 @@ stages:
         - publish: '$(Build.ARTIFACTSTAGINGDIRECTORY)\bin\Release\Publish\Zips\'
           artifact: ServiceZips
 
-  - stage: 'Publish_vPack'
+  # This final stage publishes to NuGet.org, PowerShell Gallery, vPacks, and the public symbol server. Only run it if you want to make your build outputs public!
```

Comment:
hmm; is this ACL/Perms gated? 

perhaps we can make this a two-stage or something to avoid accidentals if folks miss this commant?

Reply:
Yes it is 😄 .

It uses the ["FO-Publish-Release" environment](https://microsoft.visualstudio.com/onecore/_environments/1033?view=resources), which blocks this stage from starting until a member of the "OneCore\FO Devs" ADO group approves. T[hat group is you and your directs.](https://microsoft.visualstudio.com/OneCore/_settings/permissions?subjectDescriptor=vssgp.Uy0xLTktMTU1MTM3NDI0NS0xMTI3NzM0NDgzLTI5ODk0Mjg1NDYtMjQwNTg5MTIwNC00MTA2Nzg4NDcwLTEtMTcxNTc0NzY1OS0zNzE1NDI0MDc5LTI3MjY2OTY4NDAtMjM3MDAzODMyNg)

Reply:
granted i wont know if this works until we try it :)